cmd/wbcli/main.go

```diff
@@ -68,6 +73,7 @@ func main() {
 func registerRoutes(rs *lcd.RestServer) {
 	client.RegisterRoutes(rs.CliCtx, rs.Mux)
 	authrest.RegisterTxRoutes(rs.CliCtx, rs.Mux)
+	oraclerest.RegisterRoutes(rs.CliCtx, rs.Mux, oracle.StoreKey)
```

Comment:
Why additional register routes, while we can use AppModuleBasic.RegisterRESTRoutes? And why pass StoreKey, if store key already in module.

Reply:
Fixed